core/genesis.go

```diff
@@ -512,7 +512,7 @@ func ReadGenesisQiAlloc(filename string, logger *log.Logger) map[string]GenesisU
 
 // WriteGenesisUtxoSet writes the genesis utxo set to the database
 func AddGenesisUtxos(state *state.StateDB, nodeLocation common.Location, logger *log.Logger) {
-	qiAlloc := ReadGenesisQiAlloc("genallocs/gen_alloc_qi_"+nodeLocation.Name()+".json", logger)
+	qiAlloc := ReadGenesisQiAlloc("genallocs/gen_alloc_qi"+nodeLocation.Name()+".json", logger)
```

Comment:
This is wrong

Reply:
you looked at the outdated PR, it should be correct now